src/test/groovy/org/prebid/server/functional/CcpaSpec.groovy

```diff
@@ -1,15 +1,19 @@
 package org.prebid.server.functional
 
+import org.prebid.server.functional.model.bidder.BidderName
 import org.prebid.server.functional.model.config.AccountCcpaConfig
 import org.prebid.server.functional.model.config.AccountConfig
 import org.prebid.server.functional.model.config.AccountPrivacyConfig
 import org.prebid.server.functional.model.db.Account
+import org.prebid.server.functional.model.db.StoredRequest
+import org.prebid.server.functional.model.request.amp.AmpRequest
 import org.prebid.server.functional.model.request.auction.BidRequest
 import org.prebid.server.functional.model.request.auction.Device
 import org.prebid.server.functional.model.request.auction.Geo
 import org.prebid.server.functional.model.request.auction.RegsExt
 import org.prebid.server.functional.testcontainers.PBSTest
 import org.prebid.server.functional.util.PBSUtils
+import spock.lang.PendingFeature
 
 @PBSTest
 class CcpaSpec extends BaseSpec {
```

Comment:
Looks like this can be nested under `privacy` package as we now have 3 different groups of tests for this topic that also share a lot of code.